TLM/TLM/Manager/Impl/RoutingManager.cs

```diff
@@ -19,10 +19,10 @@ public class RoutingManager
         : AbstractGeometryObservingManager,
           IRoutingManager
     {
-        private RoutingManager() { }
-
         public static readonly RoutingManager Instance = new RoutingManager();
 
+        private RoutingManager() { }
```

Comment:
Do we need this warnings? should I move the constructor down? I think its better to get rid of the warning. It makes sense to keep the constructor close to Instance.
![image](https://user-images.githubusercontent.com/26344691/159155328-d6c32bc9-8cd2-4a33-8202-c7e54eba73fa.png)